db/keyval/etcdv3/plugin_impl.go

```diff
@@ -88,10 +92,16 @@ func (p *Plugin) Init() error {
 		return err
 	}
 
-	// register for providing status reports (polling mode)
+	// Init ETCD transport
+	p.Transport, err = p.InitTransport(p.Skeleton.Logger)
```

Comment:
Please initialize the transport in Flavor. Plugin will not be aware of the transport.